### PR TITLE
add padding to bottom line, fix left cheeto positioning and text box

### PR DESCRIPTION
**Pull request by hanlinc27**
- fix left cheeto positioning to be closer to the bottom
- fixed underline for "the bottom line" padding
- fix the padding to shrink the overall text box slightly
- added correct font for **Contact Us** button
- see Trello ticket [here](https://trello.com/c/jSgNciId)

### CODE REVIEW

File: src/components/AboutUs/TheBottomLineSection.js [hunk 1]
```diff
@@ -28,8 +28,10 @@ const useStyles = makeStyles((theme) => ({
         fontSize: '22px',
         lineHeight: '38px',
         fontWeight: 600,
+        paddingBottom: 50, 
         textDecoration: 'underline',
         textDecorationColor: '#FF7500',
+        textUnderlinePosition: 'under',
         [theme.breakpoints.down('sm')]: {
             margin: '0px 20px'
         }
```
Comment (demiwu96) [inline src/components/AboutUs/TheBottomLineSection.js:37]: On the smallest mobile screen (320px) the cheeto is not on the same line as the title, change font size to 18px to fix the problem
Screen Shot 2020-10-16 at 7 48 36 PM

**Comment by gh0stl0nely** [comment]
@rishFilet I believe there are some changes that @hanlinc27 made here to the cheetos. I am not specifically sure if it's to address the same issue but just make sure you guys are on the same page to avoid code conflicts.

File: src/components/AboutUs/TheBottomLineSection.js (hunk 1 above)
Comment (hanlinc27) [reply src/components/AboutUs/TheBottomLineSection.js:37]: thanks for the catch! fixed

**Review by demiwu96 — APPROVED** [review]

**Comment by gh0stl0nely** [comment]
@rishFilet If you think this is good since you guys are working on the same file, feel free to approve, update and merge.

**Review by rishFilet — APPROVED** [review]

**Pull request merged**